src/Infrastructure/Services/UserService.cs

```diff
@@ -183,6 +191,29 @@ public async Task<Response<NhsLoginResponse>> NhsLoginAsync(string code, string
             }
         }
 
+        private async Task SendContentfulEmailAsync(string emailName, string emailRecipient, CultureInfo selectedLocale,
+            string firstName = null)
+        {
+            if (selectedLocale == null) selectedLocale = new CultureInfo("en-GB");
```

Comment:
Generally you need to braces here to make it explicit which statements are covered by the `if`.
While leaving out the braces is valid code, it is the source of quite a few bugs.

`if (selectedLocale == null) { selectedLocale = new CultureInfo("en-GB");}`
or even better:
`selectedLocale = selectedLocale ?? new CultureInfo("en-GB");`

However, I see now that we need to set the default locale in a few different places which could be a problem.
Best to remove the check in `ContentfulService.cs` and set it once as early in the control flow as possible. Looks like here is the best place.